Objects/bytesobject.c

```diff
@@ -3640,6 +3730,25 @@ _PyBytesWriter_WriteBytes(_PyBytesWriter *writer, void *ptr,
 }
 
 
+int
+PyBytesWriter_WriteBytes(PyBytesWriter *writer, char **str,
+                         const void *bytes, Py_ssize_t size)
+{
+    if (PyBytesWriter_CheckPtr(writer, *str) < 0) {
+        return -1;
+    }
+
+    char *str2 = _PyBytesWriter_WriteBytes((_PyBytesWriter *)writer, *str,
```

Comment:
In the other functions, you used `(_PyBytesWriter*)` (no space before `*`, so I think you should fix the other places (this place is PEP 7 compliant). I'll mark those that should be amended (maybe it's better that you do CTRL+R in your IDE since I could have missed some).